plugins/source/notion/README.md

```diff
@@ -0,0 +1,35 @@
+# CloudQuery notion Source Plugin
```

Comment:
We usually put the docs under the Website, see https://github.com/cloudquery/cloudquery/tree/0c9bfbeda4aa4eb45843cac53301a50b931c1bf2/website/pages/docs/plugins/sources/alicloud for an example